kurtosis_service/kurtosis_service.go

```diff
@@ -0,0 +1,85 @@
+package kurtosis_service
+
+import (
+	"fmt"
+	"github.com/palantir/stacktrace"
+	"github.com/powerman/rpc-codec/jsonrpc2"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	kurtosisApiPort = 7443
```

Comment:
At some point will we be able to make this configurable? Or does it not matter because it's running in an isolated network inside its own container?

Reply:
It doesn't matter, and - because Docker containers are their own little isolated environments - it won't affect anything else. But we could make it configurable if we want